experimental/sdg-tracker/src/utils/constants.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Variable DCID url parameter
+ */
+export const QUERY_PARAM_VARIABLE = "v";
+
+/**
+ * Place DCID url parameter
+ */
+export const QUERY_PARAM_PLACE = "p";
+
+/**
+ * Query search text url parameter
+ */
+export const QUERY_PARAM_QUERY = "q";
+
```

Comment:
might be worth switching to use topics.

Reply:
Agreed- i'll get rid of all variable group references in a follow-up